LICENSING.md

```diff
@@ -17,3 +17,9 @@ pkg/util
 production/
 tools/
 ```
+
+The following directories and their subdirectories are licensed under their original upstream licenses:
+
+```
+public/vendor/
```

Comment:
@RichiH Guessing it's supposed to be two directories here?
```suggestion
public/
vendor/
```

Reply:
It was a copy and paste from Grafana. And I overlooked the message from way back then, sorry...

```suggestion
vendor/
```